wpeverest/everest-forms
# Fix - conditional logic field required issue

### All Submissions:






### Changes proposed in this Pull Request:



This PR fix required field issues with conditional logic

### How to test the changes in this Pull Request:

1. Create Form
2. Add conditional logic in required fields
3. Submit form without matching condition
4. Verify form works as expected

### Types of changes:




### Other information:




### Changelog entry

> Fix - conditional logic field required issue
